### PR TITLE
Create gce instance with service account scopes

### LINKED ISSUE #109: Create gce instance with permissions / service account scopes
##### Issue Type:

Feature Idea
##### Ansible Version:

ansible 1.4.3
##### Environment:

N/A
##### Summary:

Want to use gce module to create an instance with any permissions to access any other Google cloud resources like storage.  Apparently I need to pass in the [service_account_scopes flag](https://developers.google.com/compute/docs/instances)  [as is seen done in the unit test](https://github.com/ansible/ansible/blob/devel/test/gce_tests.py#L208) when calling `gcutil` directly.  How can I pass this flag using the module?
##### Steps To Reproduce:

N/A
##### Expected Results:

N/A
##### Actual Results:

N/A

cc @erjohnso

Many thanks!

NOTE: this issue was ported over from the old ansible repo where it was https://github.com/ansible/ansible/issues/6003

### PR DESCRIPTION
**Pull request by br0ziliy**
This should resolve  #109

### CODE REVIEW

**Comment by bcoca** [comment]
@erjohnso

**Comment by br0ziliy** [comment]
This pull request should implement creating GCE instances with needed "service account scopes".
This feature is quite useful, and due to GCE platform limitations it's sometimes not possible to fully automate instances provisioning (example: https://github.com/elastic/elasticsearch-cloud-gce#machine-permissions)

File: cloud/google/gce.py [hunk 1]
```diff
@@ -58,6 +58,13 @@
     required: false
     default: null
     aliases: []
+  service_account_permissions:
+    version_added: 1.5.1
+    description:
+      - service account permissions (see U(https://cloud.google.com/sdk/gcloud/reference/compute/instances/create), --scopes section for detailed information)
+    required: false
+    default: null
+    aliases: []
```
Comment (goozbach) [inline cloud/google/gce.py:67]: You may want to add the possible choices here.

Ala 

```
aliases: []
choices: ["bigquery", "cloud-platform", "compute-ro", "compute-rw", "computeaccounts-ro", "computeaccounts-rw", "datastore", "logging-write", "monitoring", "sql", "sql-admin", "storage-full", "storage-ro", "storage-rw", "taskqueue", "userinfo-email"]
```

**Comment by jimi-c** [comment]
Merged, thanks!

**Pull request merged**